<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>934</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>801</width>
     <height>290</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>290</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1980</width>
     <height>290</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 85, 255, 55);
font: 600 14pt &quot;Noto Sans&quot;;
border-radius: 10px;
margin:5px</string>
   </property>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: None;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="user_name_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);</string>
       </property>
       <property name="text">
        <string>ФИО пользователя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_login_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);
</string>
       </property>
       <property name="text">
        <string> Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_password_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);</string>
       </property>
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_prava_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);</string>
       </property>
       <property name="text">
        <string>Права</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: None;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="user_name">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_login">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_password">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);
</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user_prava">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 85, 255, 55);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
